front-end/src/components/Post/GovernanceSideBar/Proposals/ProposalVoteInfo.tsx

```diff
@@ -30,6 +33,7 @@ const ProposalVoteInfo = ({ className, proposalId }:  Props) => {
 
 		if (!apiReady) {
 			console.error('api not ready');
+			setLoadingStatus({ isLoading: false, message: 'Api not ready' });
```

Comment:
I don't think there's a need for this one. It'll be super short and doesn't bring anything to the users.